controllers/comments.controllers.js

```diff
@@ -0,0 +1,11 @@
+const { fetchCommentsByArticleId } = require('../models/comments.models')
+
+exports.getCommentsByArticleId = (req, res, next) => {
+    const { article_id } = req.params
+    fetchCommentsByArticleId(article_id).then((comments) => {
```

Comment:
Please check if the article exists to see the reason for a possible empty array of comments.